frontend/public/components/nav/_nav-header.scss

```diff
@@ -13,6 +13,7 @@
 
   .pf-c-dropdown {
     width: 100%;
+    cursor: pointer;
```

Comment:
This often indicates an accessibility issue... Is this because the existing buttons don't use a pointer cursor or another reason?

Reply:
This came about due to an [earilier comment](https://github.com/openshift/console/pull/8199#discussion_r583010755) we discussed.

Reply:
@dtaylor113 If it's a link, it should have a pointer cursor already I believe. Does it have an `href`? If not, it's an accessibility problem.